### DISCUSSION TITLE
疑似.ga域名全部挂了

### DISCUSSION THREAD

**Discussion by pencilnav** [luolongfei/freenom#227] [General]
可能是Freenom根域名服务器问题，现在基本上所有的.ga都无法解析。
有碰到这种情况的吗？
![](https://user-images.githubusercontent.com/85282140/243173001-28c6a1e9-0dd6-42eb-b40d-f4a299e77b56.png)

**Comment by luolongfei** [comment]
我没有 .ga 域名，有的可以看看

**Comment by love2wind** [comment]
据说是.ga域名被加蓬接管了。不能白嫖了
![image](https://github.com/luolongfei/freenom/assets/11996852/82ce065c-b199-4f70-870e-e03ddd6d9b34)

**Comment by pencilnav** [comment]
https://www.afnic.fr/wp-media/uploads/2023/05/ga-domain-names-soon-to-return-to-Gabonese-management-1.pdf
实锤了，被接管了。
